src/box/lua/console.lua

```diff
@@ -620,6 +623,9 @@ local function local_read(self)
             buf = buf:sub(0, #buf - #delim)
             break
         end
+        if ffi.C.sigint_is_called() == true then
+            return ''
+        end
```

Comment:
We can return `discard_buffer` (boolean) value from the `internal.readline()` function using multireturn. It'll simplify the code.

Reply:
You are right, thank you! Fixed.